codegen/gateway.go

```diff
@@ -685,7 +685,7 @@ func (e *EndpointSpec) EndpointTestConfigPath() string {
 
 // SetDownstream configures the downstream client for this endpoint spec
 func (e *EndpointSpec) SetDownstream(
-	clientModules map[string]*ClientSpec,
```

Comment:
because there is a range this doesnt require any further changes.. ffff---- golang.